modules/puzzles/module.py

```diff
@@ -52,3 +53,40 @@ def buildings_with(self, symbols):
         results = util.buildings_containing_symbols(symbols)
         self.respond('Er zijn %i gebouwen die de symbolen \'%s\' bevatten' % (len(results), symbols))
         self.respond(', '.join([str(x) for x in results]))
+
+
+class Words(TelegramModule):
```

Comment:
Do we want to have this in another class module? It is related to puzzles but perhaps another file or module would be better

Reply:
I think puzzles is best for now, its a pretty tidy module still